fl4health/clients/basic_client.py

```diff
@@ -310,18 +324,27 @@ def get_parameter_exchanger(self, config: Config) -> ParameterExchanger:
         """
         return FullParameterExchanger()
 
-    def predict(self, input: torch.Tensor) -> torch.Tensor:
+    def predict(self, input: torch.Tensor) -> Dict[str, torch.Tensor]:
         """
-        Return predictions when given input. User can override for more complex logic.
+        Return dict of str and torch.Tensor contaiing predictions when given input.
```

Comment:
containing 🙂

Reply:
Updated in [9959c89](https://github.com/VectorInstitute/FL4Health/pull/62/commits/9959c896eba8b13846f86a313be23b34619dc1cb)